lib/config/config-file.js

```diff
@@ -294,6 +295,27 @@ function write(config, filePath) {
 }
 
 /**
+ * Determines the lookup path for node packages referenced in a config file.
+ * If the config
+ * @param {string} filePath The file path to calculate lookup for.
+ * @param {string} configFilePath The config file referencing the file.
+ * @returns {string} The lookup path for the file path.
+ * @private
+ */
+function getLookupPath(filePath, configFilePath) {
+
+    // calculates the path of the project including ESLint
+    var projectPath = path.resolve(__dirname, "../../../");
```

Comment:
@ilyavolodin yes, that's what we discussed. Otherwise, nothing is going to work correctly.

@mysticatea nice catch! Need to check that.